platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheGracefulDegradationIntegrationTest.groovy

```diff
@@ -465,6 +468,49 @@ class ConfigurationCacheGracefulDegradationIntegrationTest extends AbstractConfi
         assertConfigurationCacheDegradation(true)
     }
 
+    def "can have CC incompatible tasks and requested CC degradation in the same build"() {
```

Comment:
💅  Can the test name describe the expected result rather than just the possibility?